pkgs/tools/networking/v2raya/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, fetchFromGitHub
+, mkYarnPackage
+, buildGo117Module
+, makeWrapper
+, v2ray
+, v2ray-geoip
+, v2ray-domain-list-community
+, symlinkJoin
+}:
+let
+  pname = "v2raya";
+  version = "unstable-2022-02-06";
```

Comment:
v2rayA has [releases](https://github.com/v2rayA/v2rayA/releases). Why is the unstable one used?

Reply:
The `yarn.lock` file in the latest contains some references outside the regular repository (registry.npm.com / registry.yarnpkg.com) which causes problems in mkYarnPackage. This commit is a small fix that replaces these URLs. It also contains a minor bug fix related to the v2ray icon not being properly loaded in the WebUI. All these two fixes are considered too minor to have a separate release for them.